src/main/java/com/shaft/tools/io/ProjectStructureFactory.java

```diff
@@ -0,0 +1,19 @@
+package com.shaft.tools.io;
+
+import com.shaft.cli.FileActions;
+
+import java.nio.file.Paths;
+
+public class ProjectStructureFactory {
```

Comment:
Codacy found an issue: [The utility class name 'ProjectStructureFactory' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/mohab.mohieeldeen/SHAFT_ENGINE/pullRequest?prid=6077440)